setup.py

```diff
@@ -10,7 +21,12 @@
       url='https://github.com/ZeroCater/zc_common',
       download_url='https://github.com/ZeroCater/zc_common/tarball/0.0.2',
       license='MIT',
-      packages=['zc_common'],
+      packages=get_packages('zc_common'),
+      install_requires=[
```

Comment:
Adding django/DRF as required packages means users can't install zc_common just for the timezone utils. Please remove this, and add a note to the README specifying the requirement for this particular part of the package.